package.json

```diff
@@ -1,38 +1,34 @@
 {
-  "name": "smartcar-javascript-sdk",
-  "description": "javascript auth sdk for the smartcar",
+  "name": "@smartcar/auth",
   "version": "2.0.0",
-  "author": "Smartcar <hello@smartcar.com> (https://smartcar.com)",
+  "description": "javascript auth sdk for the smartcar",
+  "main": "dist/npm/sdk.js",
   "license": "MIT",
+  "author": "Smartcar <hello@smartcar.com> (https://smartcar.com)",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/smartcar/javascript-sdk.git"
+  },
+  "bugs": {
+    "url": "https://github.com/smartcar/javascript-sdk/issues"
+  },
+  "homepage": "https://github.com/smartcar/javascript-sdk#readme",
+  "files": [
+    "dist/npm/**",
+    "doc/**"
+  ],
   "scripts": {
-    "test": "npm run build && jest",
+    "pretest": "gulp build",
+    "test": "jest",
     "posttest": "npm run lint -s",
     "lint": "eslint . --cache",
     "cover": "npm test -- --coverage",
-    "build": "gulp build",
-    "publish": "gulp publish",
+    "release": "semantic-release --branch \"$(git rev-parse --abbrev-ref HEAD)\" --extends ./build/sr-configs/local.js",
```

Comment:
This seems confusing being named `"release"`. I think something like `"plan-relase"` or `"prepare-release"` or something else to the effect that its not actually releasing would be better.